recipes/pymodsecurity/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pymodsecurity" %}
+{% set version = "0.0.1" %}
+{% set build_number = 0 %}
+{% set hash_value = "e9570494282b71172f909cb3ecfd4b7b66fa29a3141439192a48b40939893495" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_value }}
+
+build:
+  number: {{ build_number }}
+  skip: true  # [not linux]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - modsecurity
+  host:
+    - modsecurity
+    - pip
+    - python >=3.5
+    - pybind11 >=2.2
+  run:
+    - modsecurity
+    - python >=3.5
+    - pybind11 >=2.2
+
+test:
+  imports:
+    - ModSecurity
+  commands:
+    - python -c "from ModSecurity import ModSecurity"
+    - python -c "from ModSecurity import Rules"
+
+about:
+  home: https://github.com/actions-security/pymodsecurity
+  summary: Python bindings to libmodsecurity
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE.txt
```

Comment:
It would be good to packaged the ``license`` along with the package. Could you open a PR for that ?

Reply:
You mean with the package within PyPi? If so, we already did. It's schedule to be packaged in the next release.

Reply:
Ok, then. :+1: